### DISCUSSION TITLE
Recommend a "Most Popular New Movies" List?

### DISCUSSION THREAD

**Discussion by ramair02** [mza921/Plex-Auto-Collections#197] [Q&A]
I've been lookin for a trending / most popular new movies list to include as a collection, but haven't found a good list on IMDB or Trakt. Preferably a list that is regularly updated with movies released recently and that are trending, popular, well rated, etc.

Do you all have any recommendations? I'm already using IMDb Top 250, Reddit Top 250, Trakt Trending & Tautulli Most Popular. However, I would love a collection that helps my users and me discover the most popular recently released films.

**Comment by Tharic99** [comment]
Both of these have done a pretty good job for me as Radarr Lists.

https://trakt.tv/users/giladg/lists/latest-4k-releases?sort=added,asc

https://trakt.tv/users/linaspurinis/lists/top-10-pirated-movies-of-the-week-50?sort=rank,asc

**Reply by ramair02** [reply]
Thank you! Searching for Trakt lists is kind of a nightmare, so I appreciate you bringing these lists to my attention. I'm using the Top 10 Pirated Movies of the Week as a "Most Pirated This Week Collection" which is close enough to what I was looking for. Thanks much!
